### PR TITLE
feat(integration): add support for kustomer

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Added configuration for kustomer in providers.yaml
- Added kustomer to docs-v2.
- Added kustomer pages for Supported API groups in mint.json.
- Added kustomer SVG file in template-logos folder.
- Consolidate kustomer docs pages
- Added a simple sync template that retrieves a list of conversations for the organization

## Issue ticket number and link
EXT-41

## Test
This provider has been used successfully in local development to connect to Kustomer and successfully call those endpoints after authorizing using Nango.

The documentation update was reviewed using `npm run docs` and verifying on localhost.

### CODE REVIEW

**Review by khaliqgant — APPROVED** [review]

File: integration-templates/kustomer/kustomer-conversations.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                response_path: 'data',
+                limit: 100
+            }
+        };
+        for await (const conversation of nango.paginate({ ...config, endpoint })) {
+            const mappedConversation: KustomerConversation[] = conversation.map(mapConversation) || [];
+
+            const batchSize: number = mappedConversation.length;
+            totalRecords += batchSize;
+            await nango.log(`Saving batch of ${batchSize} conversation(s) (total conversation(s): ${totalRecords})`);
```
Comment (TBonnin) [inline integration-templates/kustomer/kustomer-conversations.ts]: is this `log` necessary?

**Review by TBonnin — APPROVED** [review]

File: integration-templates/kustomer/kustomer-conversations.ts (hunk 1 above)
Comment (hassan254-prog) [reply integration-templates/kustomer/kustomer-conversations.ts]: @TBonnin , no, it isn't necessary. I just removed it.

File: integration-templates/kustomer/kustomer-conversations.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                response_path: 'data',
+                limit: 100
+            }
+        };
+        for await (const conversation of nango.paginate({ ...config, endpoint })) {
+            const mappedConversation: KustomerConversation[] = conversation.map(mapConversation) || [];
+            await nango.batchSave(mappedConversation, 'KustomerConversation');
+        }
+    } catch (error) {
+        throw new Error(`Error in fetchData: ${error}`);
```
Comment (bodinsamuel) [inline integration-templates/kustomer/kustomer-conversations.ts:20]: Error stringify to {} 

```suggestion
        throw new Error(`Error in fetchData`, { cause: error });
```

**Review by bodinsamuel — APPROVED** [review]
Looks good 👌🏻

File: integration-templates/kustomer/kustomer-conversations.ts (hunk 2 above)
Comment (hassan254-prog) [reply integration-templates/kustomer/kustomer-conversations.ts:20]: @bodinsamuel just updated my pull request.

**Comment by bodinsamuel** [comment]
There are some errors in the flow `kustomer-conversations.ts(20,47): error TS2554: Expected 0-1 arguments, but got 2.`

**Pull request merged**